test/index.js

```diff
@@ -435,6 +449,8 @@ describe('function', function() {
 
       eq(S.lift3(R.reduce, [R.add], [0], [[1, 2, 3]]), [6]);
       eq(S.lift3(R.reduce, [R.add], [0], []), []);
+
+      eq(S.lift3(R.curry(area), R.dec, S.I, R.inc)(4), 6);
```

Comment:
I just left a triangle because we were talking about triangles earlier. I now see why that would be confusing :stuck_out_tongue_winking_eye: